### ISSUE TITLE
Permission Denied on Mapping Background Task

### ISSUE THREAD

**Issue by ifreer23** [mikeizbicki/cmc-csci143#246]
When running 
`$ ./src/map.py --input_path=/data-fast/twitter\ 2020/geoTwitter20-02-16.zip`
from `lambda-server:~/twitter_coronavirus (master=) $` it says
`-bash: ./src/map.py: Permission denied`. Is anyone else having this problem?

**Comment by x-ndy** [comment]
+1

**Comment by KaranGoel1** [comment]
To solve it, you need to give yourself execution permission like in the lab using the `$ chmod u+x src/map.py` command. Use a similar command for the other files as well. Hope this helps!

**Comment by ifreer23** [comment]
Thanks @KaranGoel1 !

**Comment by mikeizbicki** [comment]
@ifreer23 Your issue is hard to parse because of weird formatting.  Anytime you have a full line of text (e.g. because you are writing a bash command starting with the `$`), you need to use a code block (triple backtick) instead of inline-code (single back-tick).
